cmd/kube-controller-manager/app/options/endpointslicecontroller.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+import (
+	"github.com/spf13/pflag"
+
+	endpointsliceconfig "k8s.io/kubernetes/pkg/controller/endpointslice/config"
+)
+
+// EndpointSliceControllerOptions holds the EndpointSliceController options.
+type EndpointSliceControllerOptions struct {
+	*endpointsliceconfig.EndpointSliceControllerConfiguration
+}
+
+// AddFlags adds flags related to EndpointSliceController for controller manager to the specified FlagSet.
+func (o *EndpointSliceControllerOptions) AddFlags(fs *pflag.FlagSet) {
+	if o == nil {
+		return
+	}
+
+	fs.Int32Var(&o.ConcurrentServiceEndpointSyncs, "concurrent-service-endpoint-syncs", o.ConcurrentServiceEndpointSyncs, "The number of service endpoint syncing operations that will be done concurrently. Larger number = faster endpoint slice updating, but more CPU (and network) load.")
```

Comment:
I can't find any evidence of that being used in other cases. Since both controllers will be running in parallel I can see value in being able to configure them separately.